keyboards/adpenrose/akemipad/config.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Arturo Avila (@ADPenrose)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* Key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 5
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D4, D7, E6, B6, B4, B5 }
+#define MATRIX_COL_PINS { D3, D2, F5, F6, B2 }
+
+/* COL2ROW, ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/*Rotary encoder - set the resolution fitting your encoder.
+Most will need a value of 4. If 1 encoder click results in 2 keycodes sent
+increase the value. If you need 2 clicks for 1 keycode, decrease*/
+#define ENCODER_RESOLUTION 4
+#define ENCODERS_PAD_A { B1 }
+#define ENCODERS_PAD_B { B3 }
+#define TAP_CODE_DELAY 10
+
+/* Audio functionality */
+#define AUDIO_PIN C6
+#define AUDIO_CLICKY
+#define AUDIO_ENABLE_TONE_MULTIPLEXING
+#define AUDIO_TONE_MULTIPLEXING_RATE_DEFAULT 10
+
+// The pin connected to the data pin of the LEDs
+#define RGB_DI_PIN F4
+// The number of LEDs connected
+#define DRIVER_LED_TOTAL 27
```

Comment:
```suggestion
#define RGB_MATRIX_LED_COUNT 27
```

Reply:
Changing that line is what breaks everything. Moreover, the error message says that the problem is the undeclared DRIVER_LED_TOTAL on some quantum file. Heres an image of the error: https://imgur.com/a/sSemXZ5

Am I doing something wrong?

Reply:
Your branch is out of date, it will work when merged into current master. Commit and push and the CI build should pass.

Reply:
Ok, thanks for the heads up. I'll do it ASAP.

Reply:
Ready!

Reply:
Hmmm, seems that I need to change some other things. I'll change those too ASAP.

Reply:
There, everything's working as expected!